gulpfile.js

```diff
@@ -93,9 +86,6 @@ gulp.task('default', ['build'], function(neverEnd) {
 				files.pipe(handleDeletion())
 			)
 			.pipe(runAfterEnd(runTests));
-	}, function(err) {
-		// [TEMP] makeshift error reporting for gulp-jscs until gulp-jscs implements proper reporters
-		console.error(err.message);
```

Comment:
This callback *might* be necessary when 6to5 fails to compile due to a compilation error -- I didn't check that yet.
@jaydson can you run `npm run dev`, then put some invalid code/syntax errors and see if it will break the watch task? If it breaks the watch task, then keep this callback and just remove the comment referencing gulp-jscs. :smiley: